src/cpuinfo/SDL_cpuinfo.c

```diff
@@ -115,7 +115,7 @@
 #define CPU_CFG2_LSX  (1 << 6)
 #define CPU_CFG2_LASX (1 << 7)
 
-#if defined(SDL_ALTIVEC_BLITTERS) && defined(HAVE_SETJMP) && !defined(SDL_PLATFORM_MACOS) && !defined(SDL_PLATFORM_OPENBSD) && !defined(SDL_PLATFORM_FREEBSD)
+#if defined(SDL_ALTIVEC_BLITTERS) && defined(HAVE_SETJMP) && !defined(SDL_PLATFORM_MACOS) && !defined(SDL_PLATFORM_OPENBSD) && !defined(SDL_PLATFORM_FREEBSD) && (defined(SDL_PLATFORM_LINUX) && !defined(HAVE_GETAUXVAL))
```

Comment:
```suggestion
#if defined(SDL_ALTIVEC_BLITTERS) && defined(HAVE_SETJMP) && !defined(SDL_PLATFORM_MACOS) && !defined(SDL_PLATFORM_OPENBSD) && !defined(SDL_PLATFORM_FREEBSD) && !(defined(SDL_PLATFORM_LINUX) && defined(HAVE_GETAUXVAL))
```

Otherwise this will be disabled on all non-Linux platforms.

Reply:
> Ah, you're right - thanks! I'll fix this up in a few hours.

Done in 29d211649

Reply:
> Done in [29d2116](https://github.com/libsdl-org/SDL/commit/29d21164951e5884c98bc8d817ffeceef76b00ff)

Needs applying to release-3.2.x and possibly to SDL2 and release-2.32.x as well

Reply:
Yep, in progress.